Makefile

```diff
@@ -232,12 +232,15 @@ proto-image-push:
 	docker push $(protoImageName)
 
 ###############################################################################
-###                                Querygen                                 ###
+###                          Querygen and Modulegen                         ###
 ###############################################################################
 
 run-querygen:
 	@go run cmd/querygen/main.go
 
+run-modulegen:
+	sh ./scripts/modulegen.sh
+
```

Comment:
@p0mvn should we combine the querygen into the modulegen?

Reply:
@p0mvn I'm a bit concerned about this one. Should users pass the queries that they want in the module in `modulegen` cmd by args or  yml file?

Reply:
@p0mvn I'm a bit concerned about this one. Should users pass the queries that they want in the module in modulegen cmd by args or  yml file?

Reply:
> @p0mvn should we combine the querygen into the modulegen?

What do you mean by combine? If by that you mean to integrate querygen into module gen, then yes please.

If you mean to merge, then no

Reply:
> @p0mvn I'm a bit concerned about this one. Should users pass the queries that they want in the module in modulegen cmd by args or yml file?

So the functionality of the `run-querygen` should be preserved as is. However, if we are adding new modules via "module gen", then "module gen" should also update the "query gen" yml file so that query gen can re-generate these functions in the future

Please let me know if that makes sense

Reply:
Updated. From now `query.yml` is generated with `Params` query after `module-gen`. Users can add more queries into `query.yml` file and then run `query-gen`. The result in `x/module/client/grpc/grpc_query.go`.
@p0mvn could u take a look again thanks!